.github/workflows/Unit-Test.yml

```diff
@@ -36,18 +36,20 @@ jobs:
             ruby ../tests/commands/prop.rb && \
             ( [[ -v ALL_CHANGED_FILES ]] && [[ -n ALL_CHANGED_FILES ]] && \
             export changed_packages=$(echo ${ALL_CHANGED_FILES#packages/} | tr -d '.rb' | sort) ;\
-            [[ -n ${changed_packages-} ]] && (
-            export compatible_packages=$(comm -12 <(crew list -d compatible | sort) <(echo ${changed_packages}) | tr -s \"\n\" ' ') ; \
-            [[ -n ${compatible_packages-} ]] && (for pkg in ${compatible_packages} ;\
-            do \
-            echo \"Testing install of compatible package ${pkg}\"; \
-            yes | time crew install ${pkg}; \
+            [[ -n ${changed_packages-} ]] && ( \
+            for pkg in ${changed_packages}; \
+              do \
+            export all_compatible_packages=$(crew list -d compatible) ;\
+            if $(echo ${all_compatible_packages} | grep ${pkg}); then ;\
```

Comment:
Might want to add `| grep ^${pkg}$`.  Example packages that would throw off the results:
```
gtksourceview
gtksourceview_{3|4|5}
google_chrome
google_chrome{_beta|_stable}
```